src/utils/net/download_file.rs

```diff
@@ -13,6 +13,13 @@ use std::{
 };
 use url::Url;
 
+#[derive(Debug, Copy, Clone)]
+pub enum DownloadFileOption {
+    /// Non-resumable
+    Default,
```

Comment:
Let's be explicit. As a reader, I cannot tell what `Default` is without looking into this code.
```suggestion
    NonResumable,
```